src/lib/MiradorViewer.js

```diff
@@ -48,15 +48,22 @@ class MiradorViewer {
 
     mergedConfig.windows.forEach((miradorWindow) => {
       let thumbnailNavigationPosition;
+      let view;
       if (miradorWindow.thumbnailNavigationPosition !== undefined) {
         ({ thumbnailNavigationPosition } = miradorWindow);
       } else {
         thumbnailNavigationPosition = mergedConfig.thumbnailNavigation.defaultPosition;
       }
+      if (miradorWindow.view !== undefined) {
```

Comment:
This is intentionally left repetitive. As we add more configuration options I'm sure a pattern will emerge, but this feels still early to refactor/dry it up.